tests/actor/test_channel_registry.py

```diff
@@ -12,7 +12,7 @@
 
 async def test_channel_registry() -> None:
     """Tests for ChannelRegistry, with string as key type."""
-    reg = ChannelRegistry()
+    reg = ChannelRegistry(name="test-registry")
```

Comment:
It looks strange that you changed the way you create the registry (you changed the channel naming scheme) and you didn't have to update any tests (no tests failed). I know it is probably not worth adding a test to check the channel name, specially because it's not even exposed, but pointing it out anyway just in case.

Reply:
Not sure I understand.  ChannelRegistry takes a `name` argument now,  for the same use as the `channel_name_prefix` that you had proposed in the earlier PR.  so without this change,  the tests would fail.

Reply:
Yeah, I just meant that the name is not included in the tests. Not important.